library/kani/kani_lib.c

```diff
@@ -1,12 +1,14 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-
-#include <assert.h>
 #include <stdbool.h>
```

Comment:
While at it: do you really need/want to keep this one? I'd suggest you use `__CPROVER_bool` instead of `bool`.

Reply:
yeah, I can do that.

Reply:
Should I also use __CPROVER_size_t?

Reply:
Good idea, though you’ll still need the header for uint8_t.

Reply:
Yeah, I noticed that. I ended up just replacing the bool but I can replace size_t too.